sklearn/pipeline.py

```diff
@@ -586,9 +590,9 @@ def _transform_one(transformer, weight, X):
 def _fit_transform_one(transformer, weight, X, y,
                        **fit_params):
     if hasattr(transformer, 'fit_transform'):
-        res = transformer.fit_transform(X, y, **fit_params)
```

Comment:
If we'd add a ``and not transformer.frozen`` in the line above that would simplify ``frozen_fit`` a lot, I feel. But maybe more susceptible to bugs?